### PR TITLE
ADD: relevance + new NER apis

### PR DESCRIPTION
**Pull request by madisonmay**

### CODE REVIEW

File: lib/indico/settings.rb [hunk 1]
```diff
@@ -24,6 +27,9 @@
   "content_filtering"
 ]
 
+MULTIAPI_NOT_SUPPORTED = [
+  "relevance"
```
Comment (anneCarlson) [inline lib/indico/settings.rb]: `personas` should be in here as well.

File: spec/indico_spec.rb [hunk 2]
```diff
@@ -143,6 +144,31 @@
     expect Set.new(response.keys).subset?(Set.new(text.gsub(/\s+/m, ' ').strip.split(" ")))
```
Comment (anneCarlson) [inline spec/indico_spec.rb]: We need batch tests

File: lib/indico/settings.rb (hunk 1 above)
Comment (madisonmay) [inline lib/indico/settings.rb]: :+1:

**Comment by anneCarlson** [comment]
LGTM!

**Pull request merged**